drop-down-terminal@gs-extensions.zzrough.org/extension.js

```diff
@@ -435,9 +435,10 @@ const DropDownTerminalExtension = new Lang.Class({
     _updateWindowGeometry: function() {
         // computes the window geometry except the height
         let panelBox = Main.layoutManager.panelBox;
-        this._windowX = panelBox.x;
+        let legacyTray = Main.legacyTray.actor;
+        this._windowX = panelBox.x + legacyTray.x;
```

Comment:
Yeah you're probably right. Since this commit is almost a year old I can't be sure that there wasn't a reason behind it, but it does look weird to me too. `this._windowX = legacyTray.x;` should be enough (assuming the tray is in its default position on the bottom left).